sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientLiveTests.cs

```diff
@@ -251,6 +578,100 @@ public async Task RecognizeEntitiesBatchWithErrorTest()
             Assert.Throws<InvalidOperationException>(() => results[1].Entities.GetType());
         }
 
+        [Test]
+        public async Task RecognizeEntitiesBatchConvenienceTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            var inputs = new List<string>
+            {
+                "Microsoft was founded by Bill Gates and Paul Allen.",
+                "My cat and my dog might need to see a veterinarian."
+            };
+
+            RecognizeEntitiesResultCollection results = await client.RecognizeEntitiesBatchAsync(inputs);
+
+            foreach (RecognizeEntitiesResult result in results)
+            {
+                Assert.GreaterOrEqual(result.Entities.Count(), 1);
```

Comment:
Shouldn't this just be `Assert.Equal`, why Greater?

Reply:
I left it like this because in case the model changes and somehow it returns more or less entities than what I am currently expecting.
I do know that for sure there is one entity

Reply:
If somehow the service starts returning more than one entity when it should just one, it would be a valid bug on the service side and should be reported. So if we expect one, should we assert the same?

Reply:
Fair.
The way I see our tests is that the purpose for them is to test the SDK code, not necessarily the service, so in this case, I want to make sure that giving an input, I can serialize it, send it, and deserialize it back. 
Also, because this is AI and their models change, if they refine the model and then they are able to detect more entities from my input, my tests will fail, which doesn't necessarily mean that there is a bug in their side. No?
If we feel strongly about it, I am ok changing it, I just want to avoid having the tests failing when it isn't on us.